### PR TITLE
Gas opt and QA report

### PR DESCRIPTION
**Pull request by rfart**
rfa (warden) from code4rena

### CODE REVIEW

File: contracts/lib/Executor.sol [hunk 1]
```diff
@@ -322,9 +322,7 @@ contract Executor is Verifiers, TokenTransferrer {
         // If no conduit has been specified...
         if (conduitKey == bytes32(0)) {
             // Ensure that exactly one 721 item is being transferred.
-            if (amount != 1) {
```
Comment (axic) [inline contracts/lib/Executor.sol:325]: Removing this check seems to be not part of gas optimisation.

**Comment by 0age** [comment]
The change to Executor is not safe; can you remove that portion of this PR?

File: contracts/conduit/Conduit.sol [hunk 2]
```diff
@@ -144,11 +142,9 @@ contract Conduit is ConduitInterface, TokenTransferrer {
 
         // Iterate over each standard transfer.
         for (uint256 i = 0; i < totalStandardTransfers; ) {
-            // Retrieve the transfer in question.
-            ConduitTransfer calldata standardTransfer = standardTransfers[i];
 
             // Perform the transfer.
-            _transfer(standardTransfer);
+            _transfer(standardTransfers[i]);
```
Comment (hrkrshnn) [inline contracts/conduit/Conduit.sol:147]: I'm sceptical this would improve gas. Do you have benchmarks for this?

**Comment by 0age** [comment]
Closing as I believe all remaining changes have been committed as part of other PRs; if I missed one feel free to open a new PR with that change, or just adding yourself to `CONTRIBUTORS.md` for opening this PR!

**Pull request closed without merging**